src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Handlers/CloneContentContext.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace OrchardCore.ContentManagement.Handlers
+{
+    public class CloneContentContext : ContentContextBase
+    {
+        public ContentItem CloneContentItem { get; set; }
+        public string FieldName { get; set; } // TODO: check if this is required... Copied from O1
```

Comment:
Please find out why it's there, and if it's not used in O1 just remove it then.

Reply:
Removed it without any effect.